CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+xxxxxxxxxxxxxxxxxxx
+===================
+- Removed legacy Apify Crawler methods that are no longer supported by Apify API.
```

Comment:
shall we call it breaking change?

Reply:
We should in principle but on the other hand, it doesn't work anyway